sql/src/main/java/io/crate/analyze/TablePropertiesAnalyzer.java

```diff
@@ -118,26 +118,19 @@ public static String esToCrateSettingName(String esSettingName) {
         return (val == null) ? esSettingName : val;
     }
 
-    public void analyze(TableParameter tableParameter,
-                        TableParameterInfo tableParameterInfo,
-                        Optional<GenericProperties> properties,
-                        Row parameters) {
-        analyze(tableParameter, tableParameterInfo, properties, parameters, false);
+    public void applyDefaults(TableParameter tableParameter, TableParameterInfo tableParameterInfo) {
```

Comment:
Wouldn't expose this method. One analyze() method with the flag is more "safe" since forces the caller to correctly set the flag value.